### PR TITLE
dev/core#1664 - Fix errors recorded in log message on Manage Case scr…

### PR DESCRIPTION
**Pull request by jitendrapurohit**
…een and case dashboard page

Overview
----------------------------------------
Various errors recorded in log message on 

- Manage Case screen -> relationship tab.
- Case Dashboard Page.

Before
----------------------------------------
See [gitlab](https://lab.civicrm.org/dev/core/issues/1664) for steps to replicate the issue. 

In addition, visiting the case dashboard page records the below error in drupal logs-

>Notice: Undefined index: all in CRM_Case_Page_AJAX::getCases() (line 201 of /Users/jitendra/src/civicrm/CRM/Case/Page/AJAX.php).

After
----------------------------------------
No error message is recorded.

Comments
----------------------------------------
Gitlab - https://lab.civicrm.org/dev/core/issues/1664

### CODE REVIEW

File: CRM/Activity/Page/AJAX.php [hunk 1]
```diff
@@ -56,7 +56,7 @@ public static function getCaseGlobalRelationships() {
     // get the total row count
     CRM_Case_BAO_Case::getGlobalContacts($globalGroupInfo, NULL, FALSE, TRUE, NULL, NULL);
     // limit the rows
-    $relGlobal = CRM_Case_BAO_Case::getGlobalContacts($globalGroupInfo, $params['sortBy'], $showLinks = TRUE, FALSE, $params['offset'], $params['rp']);
+    $relGlobal = CRM_Case_BAO_Case::getGlobalContacts($globalGroupInfo, CRM_Utils_Array::value('sortBy', $params), $showLinks = TRUE, FALSE, $params['offset'], $params['rp']);
```
Comment (demeritcowboy) [inline CRM/Activity/Page/AJAX.php]: Thanks for the PR - I've noticed these messages too. FYI @colemanw might revoke your Civi card if he sees this line. Use `$params['sortBy'] ?? NULL`

File: CRM/Case/Page/AJAX.php [hunk 2]
```diff
@@ -182,7 +182,7 @@ public static function getCases() {
     $params = CRM_Core_Page_AJAX::defaultSortAndPagerParams();
     $params += CRM_Core_Page_AJAX::validateParams($requiredParameters, $optionalParameters);
 
-    $allCases = (bool) $params['all'];
+    $allCases = (bool) CRM_Utils_Array::value('all', $params, 0);
```
Comment (demeritcowboy) [inline CRM/Case/Page/AJAX.php]: `$allCases = !empty($params['all']);`

**Comment by demeritcowboy** [comment]
* General standards
    * [`r-explain`] __PASS__
    * [`r-user`] __PASS__
    * [`r-doc`] __PASS__
    * [`r-run`] __PASS__
      * Also checked the My Cases dashlet.
      * I didn't expect it to fix this, but just noting for other reviewers if you're testing sorting there is still a known separate sorting problem (https://lab.civicrm.org/dev/core/issues/1624)
* Developer standards
    * [`r-tech`] __PASS__
    * [`r-code`] __Issue__
      * The Array::value ones won't pass final review. They're so 2019.
    * [`r-maint`] __Undecided__
    * [`r-test`] __PASS__

**Comment by eileenmcnaughton** [comment]
@jitendrapurohit - @colemanw  is on the warpath against CRM_Utils_Array::value - if you swap those out this is mergeable based on the review

File: CRM/Activity/Page/AJAX.php (hunk 1 above)
Comment (jitendrapurohit) [reply CRM/Activity/Page/AJAX.php]: Thanks @demeritcowboy. Have updated the PR now.

File: CRM/Case/Page/AJAX.php (hunk 2 above)
Comment (jitendrapurohit) [reply CRM/Case/Page/AJAX.php]: done

**Comment by jitendrapurohit** [comment]
ah, sorry, wasn't aware of that. Noticed loads of warning on case screen and quickly pushed a PR with the fix. I've updated the changes and removed the usage of Utils_Array now 👍

**Comment by seamuslee001** [comment]
Merging based on @demeritcowboy 's review

**Pull request merged**